tests/installation/bootloader_hyperv.pm

```diff
@@ -0,0 +1,121 @@
+# SUSE's openQA tests
+#
+# Copyright © 2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+use base "installbasetest";
+
+use testapi;
+
+use strict;
+use warnings;
+
+use File::Basename;
+
+use Net::Telnet ();
+
+sub is_jeos() {
```

Comment:
This has already been defined at multiple places. Consider importing `lib/utils.pm` instead and you would do us a favor of cleaning up the other duplicate definitions of this helper.